src/tools/r2rdump/R2RDump.cs

```diff
@@ -405,7 +403,16 @@ private int Run(string[] args)
 
                     if (_disasm)
                     {
-                        disassembler = new Disassembler(r2r.Image, r2r.Machine);
+                        // TODO: Fix R2RDump issue where an x64 R2R image cannot be dissassembled with the x86 CoreDisTools
```

Comment:
Could you please create a github issue for this (and stick the issue number in the comment) so we don't forget about it.

Reply:
Issue is above #19564 but I can also put it in the code comment

Reply:
Yup I can add it to the comment